packages/integrations/netlify/src/index.ts

```diff
@@ -218,6 +259,7 @@ export default function netlifyIntegration(
 	let outDir: URL;
 	let rootDir: URL;
 	let astroMiddlewareEntryPoint: URL | undefined = undefined;
+	let _experimentalStaticHeaders: Map<IntegrationResolvedRoute, Headers> | undefined = undefined;
```

Comment:
It's confusing having `_experimentalStaticHeaders` and `experimentalStaticHeaders`, one of which is a flag and the other a map of the actual headers.

Reply:
What's your suggestion?

Reply:
Maybe `staticHeaderMap` and `useStaticHeaders`?